docs/source/user/api.rst

```diff
@@ -9,15 +9,69 @@ API Documentation
 API endpoints for FreeRADIUS
 ############################
 
-The following section is dedicated to API endpoints that are designed to be
-consumed by FreeRADIUS.
+The following section is dedicated to API endpoints that are designed
+to be consumed by FreeRADIUS (`Authorize`_, `Post Auth`_, `Accounting`_).
+There are three methods for request authorization:
 
-Organization API Token
-----------------------
+1. Radius User token (recommended)
+----------------------------------
+
+To use Radius User token authentication method, the
+following workflow can be used:
+
+1. User enters credentials and sends request to login in the Captive portal.
+2. The captive login page `performs authentication <#login-obtain-user-auth-token>`_ which returns **radius user token** on success.
+3. The captive login must initiate the ``POST`` request to the captive portal, using the radius user token as password, example:
+
+.. code-block:: text
+
+    curl -X POST http://localhost:8000/api/v1/authorize/ \
+         -d "username=<username>&password=<radius_token>"
+
+The ``<radius_token>`` is valid for authentication only as long
+as freeradius accounting ``Stop`` request is not sent or token
+is not deleted.
```

Comment:
I think this is incorrect, the radius token is valid for authenticaiton only once if the disposable radius token setting is true (which si the default)

Reply:
`token is not deleted.` would do it, but let me make this clearer by explicitly mentioning the flag.

Reply:
```
By default, ``<radius_token>`` is valid for authentication for one
request only. A new ``<radius_token>`` needs to be `obtained for
each request <#login-obtain-user-auth-token>`_.
However, if `OPENWISP_RADIUS_DISPOSABLE_RADIUS_USER_TOKEN
</user/settings.html#openwisp-radius-disposable-radius-user-token>`_
is set to ``False``, the ``<radius_token>`` is valid for authentication
as long as freeradius accounting ``Stop`` request is not sent
or token is not deleted.
```